firmware/controllers/engine_controller.cpp

```diff
@@ -154,14 +153,12 @@ class EngineStateBlinkingTask : public PeriodicTimerController {
 
 		if (is_running) {
 			// blink in running mode
-			enginePins.runningLedPin.setValue(counter % 2);
```

Comment:
more than one atomic PR here? looks like /s/Start/start/ occupies a big percent of "files chagnged"?

it would be great to extract trivial huge rename into separate PR